jsonObject.class.asp

```diff
@@ -30,6 +30,8 @@ const JSON_ERROR_PROPERTY_DOES_NOT_EXISTS = 3 ' DEPRECATED
 const JSON_ERROR_NOT_AN_ARRAY = 4
 const JSON_ERROR_INDEX_OUT_OF_BOUNDS = 9 ' Numbered to have the same error number as the default "Subscript out of range" exeption
 
+dim vbBack : vbBack = Chr(8)

```

Comment:
I strongly prefer to make this variable private to the `jsonObject` class, abiding to the same pattern used as other private variables there. This way we don't have to worry about global variables being overwritten or variable redeclaration errors.